### ISSUE TITLE
Isodactyly

### ISSUE THREAD

**Issue by fuzzyandblue** [obophenotype/human-phenotype-ontology#4949]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@fuzzyandblue  I am not familiar with the term isodactyly in medicine. It seems to be used in zoology to describe species whose "fingers" and "toes" are alike. Can you provide references for usages of this word in medicine?

**Comment by pnrobinson** [comment]
I do not think this term is appropriate. Please reopen this request if I am wrong and provide further information, e.g., a PubMed citation.
